sled-agent/src/profile.rs

```diff
@@ -233,7 +234,7 @@ impl Display for PropertyGroupBuilder {
             if values.len() == 1 {
                 write!(
                     f,
-                    r#"        <propval type="{ty}" name="{name}" value="{value}"/>
```

Comment:
Is this necessary? It seems odd to combine single / double quotes if we don't need to